src/allmydata/introducer/client.py

```diff
@@ -1,7 +1,14 @@
 import time
+
+# Python 2 backwards compatibility
```

Comment:
If you're not actually porting the module, I would encourage you to import just enough to get things working, and no more. Importing everything results in a bunch of potential semantic changes on Python 2. And, if you're porting the module, you'll be looking at test coverage and reading the code more carefully, but since this is a side-effect of porting some other module, better to tread lightly.